docs/tutorials/control/flutter-app.md

```diff
@@ -509,8 +526,8 @@ class _LocationScreenState extends State<LocationScreen> {
   }
 
   void _navigateToRobot(Robot robot) {
-    Navigator.of(context).push(
-        MaterialPageRoute(builder: (_) => RobotScreen(widget._viam, robot)));
+    Navigator.of(context)
+        .push(MaterialPageRoute(builder: (_) => LocationScreen(_viam, location)));
```

Comment:
Why change this? I haven't in the version I'm using. This enables one to nav from location screen to a RobotScreen()

Reply:
Ooh yeah you're right sorry I was meaning to make something more consistent but messed this part up. Reverting!